### PR TITLE
Hello react native

### PR DESCRIPTION
**Pull request by MProuts**
Completed [this tutorial](https://facebook.github.io/react-native/docs/integration-with-existing-apps) with some workarounds.

### CODE REVIEW

**Comment by MProuts** [comment]
[Stage 2: “Hello RN!”](https://trello.com/c/UkFZCF9f/1051-stage-2-hello-rn)

File: ios/NumberTileGame/Base.lproj/Main_iPhone.storyboard [hunk 1]
```diff
@@ -13,30 +18,33 @@
...
+                                <rect key="frame" x="118" y="374" width="84" height="30"/>
+                                <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
+                                <state key="normal" title="High Scores"/>
+                                <connections>
+                                    <action selector="highScoreButtonPressed:" destination="vXZ-lx-hvc" eventType="touchUpInside" id="c41-Gs-kwC"/>
+                                </connections>
+                            </button>
                         </subviews>
-                        <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="calibratedWhite"/>
+                        <color key="backgroundColor" red="1" green="1" blue="1" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
```
Comment (jpendry) [inline ios/NumberTileGame/Base.lproj/Main_iPhone.storyboard:43]: Are we sure about this?

**Review by jpendry — APPROVED** [review]
👍

File: ios/NumberTileGame/Base.lproj/Main_iPhone.storyboard (hunk 1 above)
Comment (jpendry) [reply ios/NumberTileGame/Base.lproj/Main_iPhone.storyboard:43]: See: https://twitter.com/blowdart/status/1185375804613488640

**Comment by MProuts** [comment]
I accidentally kept pushing commits to this branch after Jimmy reviewed.  I moved the part 2 commits [here](https://github.com/logcheck/iOS-2048/pull/2), which I will merge given Jimmy's 👍.

I'll open a separate PR for the part 3 commits.

**Pull request closed without merging**